packages/api/src/external/aws/app-config.ts

```diff
@@ -107,6 +107,10 @@ export async function getCxsWithNoWebhookPongFeatureFlagValue(): Promise<string[
   return getCxsWithFeatureFlagEnabled("cxsWithNoWebhookPongFeatureFlag");
 }
 
+export async function getCxsWithEpicEnabled(): Promise<string[]> {
+  return getCxsWithFeatureFlagEnabled("cxsWithEpicEnabled");
+}
```

Comment:
Could we use the other way around, like the FF has the list of IDs that are disabled to use Epic? This should be a much smaller list. 🙏

Reply:
Unfortunately not, since the default state is "disabled until approved".

⚠️  Speaking of which, we'll need to add updating this FF in prod to the release plan so cxs do not have outages

Reply:
Taken to Slack